src/app/project-overview/component.ts

```diff
@@ -109,6 +118,10 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
     component.projectId = this.project.id;
     component.showQuotaWidgetDetails = true;
     component.showIcon = false;
+
+    this._projectChange.pipe(takeUntil(this._unsubscribe)).subscribe(_ => {
+      component.projectId = this.project.id;
```

Comment:
Should we implement `ngOnChanges` in quota-widget component so that it immediately reloads quotas if `projectId` is changed?

Reply:
Alright